### PR TITLE
Seri skyculture

### PR DESCRIPTION
**Pull request by gzotti**
### Description

This is a contribution from Raúl Pérez-Enríquez who provided these after the Oxford-12 conference. 
The files may need some additional reformatting and improvements until early December, then merged to allow translation. Especially, the descriptions of the constellations should consist of a little image on the left and text on the right. Before the next heading, the vertical space given by excessive image height should be cleared. I only edited the English file. The Spanish version is his original and should likewise be reformatted. 

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] This change requires a documentation update

### How Has This Been Tested?

**Test Configuration**:
* Operating system: Windows 10

## Checklist:

- [x] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (header file)
- [ ] I have updated the respective chapter in the Stellarium User Guide
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by gzotti** [comment]
@alex-w  Where is the list of regions for the regionparam_**e**_ter? It is not yet in the SUG!

**Comment by alex-w** [comment]
@gzotti please check the master

**Review by alex-w — COMMENTED** [review]
The third and second columns in constellation_names file should be updated:

`007 "Yay_xapi_hanopas" _("Yay xapi hanopas (Young hunter)")` -> `007 "Yay xapi hanopas" _("Young hunter")`

**Comment by alex-w** [comment]
But I meant the data, not white spaces

**Comment by gzotti** [comment]
Ah yes, ok. The second column should have underscores replaced by space.  But the third, `_("name (translation)")` is what we describe in SUG:9.3.1.

**Comment by alex-w** [comment]
> Ah yes, ok. The second column should have underscores replaced by space. But the third, `_("name (translation)")` is what we describe in SUG:9.3.1.

Let me quote: 
```
The native constellations are listed in
\file{constellation\_names.eng.fab}. It consists of 3 simple columns:
Abbreviation (or just a serial number), native name, and English
translation. The writing \texttt{\_("name")} allows automatic
translation of the English strings to other languages. These strings
will be used as constellation labels.
```

**Comment by gzotti** [comment]
I refer to:
> If your skyculture is not a variant of the generally known Western skyculture, please include an
> English translation to the name given in the native language. Else translators will not be able to
> translate the name. See a good example in the Mongolian skyculture.

As long as we have not changed the display code and allow multiple composition variants like "native+(translation)", "native_glyphs+[transliteration]+(translation)" etc., this translatable bi-lingual coding as "native (english_translation)" is most probably the best format, as most users will want to see native name plus translation at the same time. The Seri users will just use the "native" setting. If the Abbreviation column would use memorizable abbreviations, some Seri could use them and have less display clutter.

Where I need technical help, however, is the current bad display of the description.en.utf8. The figures are left of the text, but if there is not enough text, the next heading is already displayed right of the previous image. This is ugly. Is there a way in HTML to "clear space" before next heading of the same value (`<h3>`), apart from reformatting all into a table?

**Comment by alex-w** [comment]
I think the data from **external links** should be defined in `references.fab` file

**Comment by alex-w** [comment]
Probably you are right and table will be best solution here. Shall I do it?

**Comment by gzotti** [comment]
Yes please. TNX, G.

**Comment by alex-w** [comment]
Done, please check the results

**Comment by sushoff** [comment]
- I did not read the spanish version (won't have the time now) but checked it in case of a typo correction
- for the star names: I recommend to delete all standard names (like Deneb Kaitos, Rigel...) and add only the names from the very culture 
- the image [quetoh.png](https://github.com/Stellarium/stellarium/pull/2863/files#diff-a2976c1423ff9a95985c562e099692ba38956f9be3b66107dc77d46054e94b04) needs to be reworked because the background is not tidied up

I think, the rest is ok

**Comment by raulpe55** [comment]
I am in the project. How can I see the files and make changes?

File: skycultures/seri/description.en.utf8 [hunk 1]
```diff
@@ -111,11 +111,11 @@ Quetoh yacaso or 'spell of the dove' responsible of eating that fruit.</p></td>
 <tr>
 <td><img src="yay_xapi.png" height="100" /></td>
 <td><p>The young hunter, says the legend, was a boy who went hunting and several weeks after his departure, 
-he was still absent and the members of his camp gave him as dead. But several months later, at daybreak,
```
Comment (raulpe55) [inline skycultures/seri/description.en.utf8]: remove the comment

Comment (raulpe55) [inline skycultures/seri/description.en.utf8]: remove the comment

File: skycultures/seri/description.en.utf8 [hunk 2]
```diff
@@ -111,11 +111,11 @@ Quetoh yacaso or 'spell of the dove' responsible of eating that fruit.</p></td>
 <tr>
 <td><img src="yay_xapi.png" height="100" /></td>
 <td><p>The young hunter, says the legend, was a boy who went hunting and several weeks after his departure, 
-he was still absent and the members of his camp gave him as dead. But several months later, at daybreak, 
-he shown himself to his parents who were surprised telling him: "We think you have daid, we have many days 
+he was still absent and the members of his camp considered him dead. But several months later, at daybreak,
```
Comment (raulpe55) [inline skycultures/seri/description.en.utf8:135]: THIS IS ok

File: skycultures/seri/description.en.utf8 [hunk 3]
```diff
@@ -111,11 +111,11 @@ Quetoh yacaso or 'spell of the dove' responsible of eating that fruit.</p></td>
 <tr>
 <td><img src="yay_xapi.png" height="100" /></td>
 <td><p>The young hunter, says the legend, was a boy who went hunting and several weeks after his departure, 
-he was still absent and the members of his camp gave him as dead. But several months later, at daybreak, 
-he shown himself to his parents who were surprised telling him: "We think you have daid, we have many days 
+he was still absent and the members of his camp considered him dead. But several months later, at daybreak, 
+he has shown himself to his parents who were surprised telling him: "We thought you were dead, we have spent many days
```
Comment (raulpe55) [inline skycultures/seri/description.en.utf8:136]: This is OK

File: skycultures/seri/description.en.utf8 [hunk 4]
```diff
@@ -111,11 +111,11 @@ Quetoh yacaso or 'spell of the dove' responsible of eating that fruit.</p></td>
 <tr>
 <td><img src="yay_xapi.png" height="100" /></td>
 <td><p>The young hunter, says the legend, was a boy who went hunting and several weeks after his departure, 
-he was still absent and the members of his camp gave him as dead. But several months later, at daybreak, 
-he shown himself to his parents who were surprised telling him: "We think you have daid, we have many days 
+he was still absent and the members of his camp considered him dead. But several months later, at daybreak, 
+he has shown himself to his parents who were surprised telling him: "We thought you were dead, we have spent many days 
 looking for you, until we were tired; we cried for you a lot". The young man told his parents: "I live 
 near the beach, I found a good place and stayed there". "But what kind of water you drink?" They asked. 
-"I found any water and that is I drink, Haxhaxaha tahomisi", he said them while talking about the Tecomate
```
Comment (raulpe55) [inline skycultures/seri/description.en.utf8]: remove comment

File: skycultures/seri/description.en.utf8 [hunk 5]
```diff
@@ -111,11 +111,11 @@ Quetoh yacaso or 'spell of the dove' responsible of eating that fruit.</p></td>
...
 <td><img src="yay_xapi.png" height="100" /></td>
 <td><p>The young hunter, says the legend, was a boy who went hunting and several weeks after his departure, 
-he was still absent and the members of his camp gave him as dead. But several months later, at daybreak, 
-he shown himself to his parents who were surprised telling him: "We think you have daid, we have many days 
+he was still absent and the members of his camp considered him dead. But several months later, at daybreak, 
+he has shown himself to his parents who were surprised telling him: "We thought you were dead, we have spent many days 
 looking for you, until we were tired; we cried for you a lot". The young man told his parents: "I live 
 near the beach, I found a good place and stayed there". "But what kind of water you drink?" They asked. 
-"I found any water and that is I drink, Haxhaxaha tahomisi", he said them while talking about the Tecomate 
+"I found any water and that is I drink, Haxhaxaha tahomisi", he said to them while talking about the Tecomate
```
Comment (raulpe55) [inline skycultures/seri/description.en.utf8:139]: This is OK

File: skycultures/seri/description.en.utf8 [hunk 6]
```diff
@@ -126,7 +126,7 @@ by a conflict with his elder brother. It appears directly towards the south.</p>
 <p>All this knowledge of Comca'ac people comes from prehispanic epoch, and it has prevailed through 
 centuries, thanks to oral tradition and the strength of the community to colonization, at first, and his 
 opposition to evangelization, later on. Still today, it is possible to hear the stories and legends 
-behind the naming of Constellations and the events on the sea and on the desert. With the recovery of 
+behind the naming of constellations and the events on the sea and on the desert. With the recovery of
```
Comment (raulpe55) [inline skycultures/seri/description.en.utf8:150]: this change i OK

**Review by raulpe55 — APPROVED** [review]

**Comment by sushoff** [comment]
please also clear the backgrounds of the images 

- https://github.com/Stellarium/stellarium/pull/2863/files/291770b04f90d0dc88d75340fa45422c06950dd4#diff-704f9ee9420c3e30a57befead71fbc6fee43bd1dd28855d5e7db10d0a2f43429

- https://github.com/Stellarium/stellarium/pull/2863/files/291770b04f90d0dc88d75340fa45422c06950dd4#diff-78bc672b3dfbea0a0dc6be06563d58072d83f9d8caa456477863c969a0443cee

The white rubbish-dots in the backgrounds of images will disturb the view of the star chart when applied. 

This one is ok (not perfect but ok) but could be improved if time allows 

- https://github.com/Stellarium/stellarium/pull/2863/files/291770b04f90d0dc88d75340fa45422c06950dd4#diff-a2976c1423ff9a95985c562e099692ba38956f9be3b66107dc77d46054e94b04

File: skycultures/seri/description.en.utf8 [hunk 7]
```diff
@@ -11,7 +11,7 @@
  <li>the Tepocaj, the fishermen, who inhabited the region between Puerto Libertad y Puerto Peñasco; and, </li>
  <li>the Tosni hox hanoquipilc, who fed themselves on sea birds (ducks and pelicans), inhabited Island San Lorenzo.</li>
 </ul></p>
-<p> Nowadays, the Seri people live at Punta Chueca and Desemboque, located in the coast of the northwestern state of Sonora, M&#233;xico.  
+<p><img src="abuela.png" width="250" align="left" />Nowadays, the Seri people live at Punta Chueca and Desemboque, located in the coast of the northwestern state of Sonora, M&#233;xico.
```
Comment (alex-w) [inline skycultures/seri/description.en.utf8:14]: Mexico - shouldn’t be diacritics in English term

File: skycultures/seri/description.en.utf8 [hunk 8]
```diff
@@ -60,37 +60,40 @@ around two months later, Comca'ac people returned to the beach carrying with the
...
 <tr>
 <td><img src="haapjc.png" height="100" /></td>
 <td><p>This constellation is, perhaps, the most complex of Comca'ac ones, due to the fact that it 
 covers a wide area in the sky and in it, one can find stars from Taurus, Twins, and the Capella star. 
-It can be said that Hapjc (Deer hunter) has: the deer (white tail, bura and bighorn, or Hépem, 
-Háap and Mójit, respectively) which are the stars known as Hassaleh, Elnath and 123 Tau; the 
-spear with its flint at Aldebaran and the stars forming a 'V' of Taurus, while the feathers are 
-Castor and Pollux of Gemini. The hunter is Cozaactim ("Who crosses"), properly said, his head is 
-Capella.</p>
+It can be said that Haapjc (Deer hunter) has the following sections: the whitetail deer, the bura deer and
```
Comment (alex-w) [inline skycultures/seri/description.en.utf8:80]: “ bura” is Spanish term probably

File: skycultures/seri/description.en.utf8 [hunk 9]
```diff
@@ -60,37 +60,40 @@ around two months later, Comca'ac people returned to the beach carrying with the
...
-Capella.</p>
+It can be said that Haapjc (Deer hunter) has the following sections: the whitetail deer, the bura deer and 
+bighorn deer (Hépem, Háap and Möjit, respectively), which are the stars known as Hassaleh, Elnath and 123 Tau; 
+the spear with its flint at Aldebaran and the stars forming a 'V' of Taurus, while the feathers are 
+Castor and Pollux of Gemini; and, finally, the hunter, properly said, is Cozaactim ("Who crosses"), 
+with his head at Capella.</p>
 <p>The legend says that Cozaactim threw his spear and hit the Bura deer. Háap was wounded and flew 
 through the sky bleeding; then, a little drop of blood fell in the Tiburón Island, and that is the reason 
-why bura deer are native of the island. This constellation can be observed in September morning sky.</p></td>
+why bura deer are native of the island. This constellation can be observed in September morning skies.</p></td>
```
Comment (alex-w) [inline skycultures/seri/description.en.utf8:87]: The “bura” term again

File: skycultures/seri/description.en.utf8 [hunk 10]
```diff
@@ -60,37 +60,40 @@ around two months later, Comca'ac people returned to the beach carrying with the
...
+
 <tr><td colspan="2"><h3><notr>Hácosa</notr></h3></td></tr>
 <tr>
 <td><img src="hacosa.png" height="100" /></td>
-<td><p>It is said that when this constellation is pointing to the south, it appears as the instrument Hácosa 
-(Pitahaya-cutter), and it is time to collect the wild fruits: péchitas of mezquite and pitahayas (cactus fruit), 
-for example. When there is no sign of it, the season will be bad.</p></td>
+<td><p>It is said that when this constellation is pointing to the top of the sky, at September 7th, 
+it appears similar to the instrument Hácosa (Pitahaya-cutter), and it is time to collect the wild fruits: 
+péchitas of mezquite (Mezquite seeds) and pitahayas (cactus fruit), for example.
```
Comment (alex-w) [inline skycultures/seri/description.en.utf8:95]: “péchitas of mezquite” - Spanish-English mix?

File: skycultures/seri/description.en.utf8 (hunk 9 above)
Comment (gzotti) [reply skycultures/seri/description.en.utf8:87]: Ah, sorry!

File: skycultures/seri/description.en.utf8 (hunk 8 above)
Comment (gzotti) [reply skycultures/seri/description.en.utf8:80]: Seems to be a proper name.

**Pull request merged**